backend/src/backend/primary/routers/general.py

```diff
@@ -47,12 +52,25 @@ async def logged_in_user(request: Request) -> UserInfo:
 
     user_info = UserInfo(
         username=authenticated_user.get_username(),
+        avatar_b64str=None,
+        display_name=None,
         has_sumo_access=authenticated_user.has_sumo_access_token(),
         has_smda_access=authenticated_user.has_smda_access_token(),
     )
 
-    return user_info
+    if includeAvatar:
+        headers = { "Authorization": f"Bearer {authenticated_user.get_graph_access_token()}" }
+        async with httpx.AsyncClient() as client:
+            result = await client.get("https://graph.microsoft.com/v1.0/me/photo/$value", headers=headers)
+            if result.status_code == 200:
+                user_info.avatar_b64str = base64.b64encode(result.content)
+        
+        async with httpx.AsyncClient() as client:
+            result = await client.get("https://graph.microsoft.com/v1.0/me", headers=headers)
+            if result.status_code == 200:
+                user_info.display_name = json.loads(result.content.decode("utf-8")).get("displayName", None)
```

Comment:
I don't know `httpx` enough yet to immediately know the answer to this: Are these two calls done sequentially here? If they are, and since they are independent, we could perhaps do them in parallell and just `await` until both are done before `returning`? Should speed up the `logged_in_user` response time in that case.